lib/segment/src/index/plain_payload_index.rs

```diff
@@ -301,6 +302,16 @@ impl VectorIndex for PlainIndex {
     fn update_vector(&mut self, _id: PointOffsetType, _vector: VectorRef) -> OperationResult<()> {
         Ok(())
     }
+
+    fn set_quantized_vectors(
```

Comment:
If this function is called,  `quantized_vectors` is always `Some`. It's possible to configure qdrant in some way to trigger this function (because we can use plain index for non-appendable segments and configure quantization simultaneously). I find it a correct behavior and propose to do nothing. I can add support of quantization for plain index